src/terminal/message.rs

```diff
@@ -44,3 +44,8 @@ pub fn help(msg: &str) {
     let msg = format!("{} {}", emoji::SLEUTH, msg);
     message(&msg);
 }
+
+pub fn deprecation_warning(msg: &str) {
+    let msg = format!("\n\t{} {}", emoji::WARN, msg);
+    message(&msg);
+}
```

Comment:
ooh we should make this use #1128 big neon box

Reply:
yeah i knew that was incoming; comes down to which PR gets merged first 😜

Reply:
maybe we can make it look different - red border and double border >:)

Reply:
sure; which pr gets merged first tho will determine who gets to do it 😛

Reply:
did this.